### PR TITLE
Update rubygem-foreman_wreckingball to 5.0.0

### PR DESCRIPTION
**Pull request by laugmanuel**

### CODE REVIEW

File: packages/plugins/rubygem-foreman_wreckingball/rubygem-foreman_wreckingball.spec [hunk 1]
```diff
@@ -22,6 +22,7 @@ Requires: %{?scl_prefix_ruby}ruby(release)
 Requires: %{?scl_prefix_ruby}ruby
 Requires: %{?scl_prefix_ruby}ruby(rubygems)
 Requires: %{?scl_prefix}rubygem(foreman-tasks) >= 0.13.1
+Requires: %{?scl_prefix}rubygem(foreman_puppet)
```
Comment (ekohl) [inline packages/plugins/rubygem-foreman_wreckingball/rubygem-foreman_wreckingball.spec:25]: I wonder why this isn't generated as a `BuildRequires`, but it's causing the build failure. Did you use [bump_rpm.sh](https://github.com/theforeman/foreman-packaging/blob/rpm/develop/bump_rpm.sh)?

**Review by ekohl — CHANGES_REQUESTED** [review]

File: packages/plugins/rubygem-foreman_wreckingball/rubygem-foreman_wreckingball.spec (hunk 1 above)
Comment (laugmanuel) [reply packages/plugins/rubygem-foreman_wreckingball/rubygem-foreman_wreckingball.spec:25]: The script doesn't work on MacOS and therefore I needed to do the steps manually. I'll have another try..

Comment (laugmanuel) [reply packages/plugins/rubygem-foreman_wreckingball/rubygem-foreman_wreckingball.spec]: I've regenerated the spec-file using gem2rpm with the default template. Can we retry the build?

**Comment by laugmanuel** [comment]
[test]

**Comment by laugmanuel** [comment]
@ekohl can you retrigger the build pipeline?

File: packages/plugins/rubygem-foreman_wreckingball/rubygem-foreman_wreckingball.spec [hunk 2]
```diff
@@ -1,82 +1,51 @@
...
+Name: rubygem-%{gem_name}
+Version: 5.0.0
+Release: 1%{?dist}
 Summary: Adds status checks of the VMWare VMs to Foreman
-Group: Applications/Systems
-License: GPLv3+
+License: GPLv3
 URL: https://github.com/dm-drogeriemarkt/foreman_wreckingball
-Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
+Source0: %{gem_name}-%{version}.gem
```
Comment (evgeni) [inline packages/plugins/rubygem-foreman_wreckingball/rubygem-foreman_wreckingball.spec:10]: This is breaking the build

File: packages/plugins/rubygem-foreman_wreckingball/rubygem-foreman_wreckingball.spec [hunk 3]
```diff
@@ -87,20 +56,17 @@ cp -pa .%{gem_dir}/* \
...
-%{foreman_assets_foreman}
 
 %files doc
 %doc %{gem_docdir}
 %doc %{gem_instdir}/README.md
 %{gem_instdir}/Rakefile
 %{gem_instdir}/test
 
-%posttrans
-%{foreman_plugin_log}
```
Comment (evgeni) [inline packages/plugins/rubygem-foreman_wreckingball/rubygem-foreman_wreckingball.spec:101]: please don't remove that

File: packages/plugins/rubygem-foreman_wreckingball/rubygem-foreman_wreckingball.spec (hunk 2 above)
Comment (laugmanuel) [reply packages/plugins/rubygem-foreman_wreckingball/rubygem-foreman_wreckingball.spec]: Was already at fixing it. I've used gem2rpm and the provided template. Not sure why the URL got truncated.

File: packages/plugins/rubygem-foreman_wreckingball/rubygem-foreman_wreckingball.spec (hunk 3 above)
Comment (laugmanuel) [reply packages/plugins/rubygem-foreman_wreckingball/rubygem-foreman_wreckingball.spec]: Already fixed

**Review by evgeni — APPROVED** [review]

**Pull request merged**